examples/embed.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2022 The KubeOne Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package examples
+
+import (
+	"embed"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+//go:embed terraform/*
+var terraformFS embed.FS
+
+func CopyTo(dst, src string) error {
+	return fs.WalkDir(terraformFS, src, func(fspath string, de fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
```

Comment:
In our case it's a NOOP basically, since original `fs.WalkDirFunc` is for general case of working with fs.FS, not only embedded. But our embedded `FS` would always pass by this input err. Having `err` is absolutely not an expected thing. That's why `return err` regardless, to signal that something unimaginable happened.